### DISCUSSION TITLE
pricing/free tier plan description for databases

### DISCUSSION THREAD

**Discussion by stardvst** [neondatabase/neon#3285] [Q&A]
i want to use postgres database. i couldn't find how much is the free tier, and when the paid plan starts.
please point the pricing plans for using the databases (after the free tier).
thanks.

**Comment by danieltprice** [comment]
Thank you for your question. Paid plan pricing can be found on the Pricing page on our website: https://neon.tech/pricing
More details about pricing are available on the Billing page in our documentation: https://neon.tech/docs/introduction/billing
